### PR TITLE
Extend network test to perform upload/download

### PR DESCRIPTION
**Pull request by meatballhat**
and stub in a separate script to exercise network/disk specifically related to
how build caching works

### CODE REVIEW

File: lib/chirp/scripts/cache [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+main() {
+  # TODO
```
Comment (sarahhodne) [inline lib/chirp/scripts/cache]: Is this PR waiting for this to be implemented, or is the intention to merge this with the TODO?

File: lib/chirp/scripts/network [hunk 2]
```diff
@@ -26,10 +33,26 @@ main() {
...
   for url in "${URLS[@]}"; do
-    curl -fsSL "$url" &
+    curl -o /dev/null -fsSL "$url" &
   done
 
   wait
 }
 
+___down_up_test() {
+  [[ "${CHIRP_DOWN_UP_URL}" ]] || {
```
Comment (sarahhodne) [inline lib/chirp/scripts/network:43]: Is `CHIRP_DOWN_UP_URL` meant to be set in this PR?

Comment (meatballhat) [reply lib/chirp/scripts/network:43]: No, this is meant to be provided at the level of a repo that uses this gem.

File: lib/chirp/scripts/cache (hunk 1 above)
Comment (meatballhat) [reply lib/chirp/scripts/cache]: I'll drop out the `cache` script until I'm able to properly implement it :+1:

**Review by sarahhodne — APPROVED** [review]

**Pull request merged**